src/Features/Core/Portable/Wrapping/SeparatedSyntaxList/SeparatedSyntaxListCodeActionComputer.cs

```diff
@@ -132,8 +133,8 @@ private string GetBraceTokenIndentation()
             {
                 var previousToken = _listSyntax.GetFirstToken().GetPreviousToken();
 
-                // Block indentation is the only style that correctly indents across all initializer expressions
-                return GetIndentationAfter(previousToken, Formatting.FormattingOptions.IndentStyle.Block);
+                // Block indentation is the only style that correctly indents across all initializer expressions                
```

Comment:
Whitespace at end of line.

Reply:
Somehow our code style validation did not catch this.

Reply:
Odd I don't see it in the editor :-0. Maybe some gihub bug.